README.md

```diff
@@ -1,3 +1,181 @@
-# Locker
+<!-- ABOUT THE PROJECT -->
+## About The Project
 
-WIP
+[![Product Name Screen Shot][product-screenshot]](https://example.com)
+
+
+
+Use the `BLANK_README.md` to get started.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+
+### Built With
+
+This section should list any major frameworks/libraries used to bootstrap your project. Leave any add-ons/plugins for the acknowledgements section. Here are a few examples.
+
+* [![Next][Next.js]][Next-url]
+* [![React][React.js]][React-url]
+* [![Vue][Vue.js]][Vue-url]
+* [![Angular][Angular.io]][Angular-url]
+* [![Svelte][Svelte.dev]][Svelte-url]
+* [![Laravel][Laravel.com]][Laravel-url]
+* [![Bootstrap][Bootstrap.com]][Bootstrap-url]
+* [![JQuery][JQuery.com]][JQuery-url]
```

Comment:
Add actual frameworks being used.

- Next
- React
- Prisma
- tRPC
- Tailwindcss
- Radix UI

Reply:
Removed it as it didn't go well with the design